### ISSUE TITLE
Feature Request: Better tracker for feature requests

### ISSUE THREAD

**Issue by spiritloveroot** [rarfell/dimeApp#9]
A website like canny.io or something similar where it's easier for users to see what feature requests are most popular and an ability to add one's vote to already existing feature requests, etc...

**Comment by jeffrey-zang** [comment]
Good idea! Maybe something like the [Obsidian Forum](https://forum.obsidian.md/) or a GitHub discussions page.

**Comment by Rodaviva29** [comment]
> Good idea! Maybe something like the [Obsidian Forum](https://forum.obsidian.md/) or a GitHub discussions page.

Hey Jeff,

Sending to you some recommendations for it.

# Self hosted options

I get used to https://feedbacky.net/, that is not very well known, but had all the features I needed. They said they would do a overhaul anytime soon because it has been inactive for a lot of time now.

Got to know recently about https://logchimp.codecarrot.net/, that seems to have a lot similarity with Feedbacky and there development active.

This two are awesome to community active contributions in feedback and also we (the community) can keep track of your roadmap and changelog at the same place. 

There is also Fider (https://fider.io/) that is much more popular and well known, but I personally don't like because it only have the voting tool feature and lacks roadmap/changelog.

Cloud hosted options

- https://userjot.com/, absolutely gorgeous (launched recently) and has a free plan.

**Comment by jeffrey-zang** [comment]
Thanks, will take a look! We aren't a super big community, so using GitHub discussions is probably easiest :D

**Comment by jeffrey-zang** [comment]
Actually I'm really loving UserJot! Will use that.

**Comment by Rodaviva29** [comment]
> Thanks, will take a look! We aren't a super big community, so using GitHub discussions is probably easiest :D

But we can go (and dream) biggg! 💥💥 :P

> Actually I'm really loving UserJot! Will use that.

Ahaha, when I recommend UserJot they always fall in love with the design, I think it aligns very well with the newest design choices.

**Comment by jeffrey-zang** [comment]
Userjot has been made: https://dimeapp.userjot.com/board/all

Will start transferring all GitHub issues to there.

**Comment by jeffrey-zang** [comment]
Opening this back up because I'm not totally sold on UserJot.

**Comment by Rodaviva29** [comment]
> Opening this back up because I'm not totally sold on UserJot.

I get it, I forgot that big part, integration with Linear/GitHub would be better to make a move. Probably for now Discussions would be the best solution! Either way, I emailed the Creator of User Jot asking for a Personalized Plan for Open Source Projects and got a positive reply to make an exception! Can you share your email so I can add you into the email response? Maybe to make some suggestion came to reality faster :)

**Comment by Rodaviva29** [comment]
> doesn't have GitHub or Linear integration, makes it hard to track tasks + link to PRs
> most ppl already have a GH account, but not a UserJot account, so it increases friction of discussions

Supporting GitHub login would very good as well, saw your suggestions about the Integrations will make one for GH Sign in. Also, if DimeApp could get a Special Plan for being Open-Source and without financial plans, Guest Mode would also a possibility to get around this.

**Comment by jeffrey-zang** [comment]
Yep, sounds good. My email is hi@jeffz.dev

**Comment by Rodaviva29** [comment]
> Yep, sounds good. My email is [hi@jeffz.dev](mailto:hi@jeffz.dev)

Sent :)
